<launch>  
    <node pkg="bitten" type="wp_node" name="wp_node"/>

    <node pkg="bitten" type="commander_node" name="commander_node"/>

    <node pkg="bitten" type="movement_node" name="movement_node"/>
</launch>